website/docs/expansion.md

```diff
@@ -0,0 +1,223 @@
+---
+id: expansion
+title: Early Rejection of Generator Resources
+---
+
+> 🚧 This feature is in _alpha_ stage, and is not enabled by default. To
```

Comment:
```suggestion
> ❗This feature is in _alpha_ stage, and is not enabled by default. To
```
Can you also add the `Feature State` like https://github.com/open-policy-agent/gatekeeper/pull/2260/files#diff-953d3a7e7497c49d0d940dec88438549eb9fff09afa48eb261123ab3773a9290R6-R8

Reply:
nit. can you update to ❗so it's consistent with other features still in alpha stage?